src/utils/validations.ts

```diff
@@ -330,3 +330,27 @@ export const validateOutputs = (
     'output'
   )
 }
+
+/**
+ * It validates that an ABI have a specific function
+ * @param {string} functionName - the name of the function
+ * @param {string} paramsTypes - the types of the params of the function
+ * @param {Interface} abi - Interface - the abi to validate
+ */
+export const implementsFunction = (
+  functionName: string,
+  paramsTypes: string[] | null | undefined,
```

Comment:
I think it's weird to pass null or undefined as a second parameter and a third one being required. It could be having the Abi as the first one, the second one the function name and the third one an optional `paramTypes?: string[]`